llvm/lib/Support/APFloat.cpp

```diff
@@ -68,6 +68,10 @@ enum class fltNonfiniteBehavior {
   // `fltNanEncoding` enum. We treat all NaNs as quiet, as the available
   // encodings do not distinguish between signalling and quiet NaN.
   NanOnly,
+
+  // This behavior is present in Float6E3M2FN and Float6E2M3FN types.
+  // There is no representation for Inf or NaN.
+  NoNanInf,
```

Comment:
nit:
```suggestion
  // This behavior is present in Float6E3M2FN and Float6E2M3FN types,
  // which do not support Inf or NaN values.
  NoNanInf,
```
also, is there a standard name for such FP types? Maybe `Finite` or would that be too overloaded?

Reply:
Invert and call SupportsNonFinite?

Reply:
Updated the comment in the latest revision.

For the naming:
The MX spec has a few other types but they support either Inf or Nan. So, I could not name it generically like "MXType".

I see that "inverting" (and naming) would change the condition-checks in the code.
Looking at the places where this is being used (in OR with other conditions), it gives better readability if we do not invert.

So, below are a few options that I could think of:
1) Finite
2) FiniteOnly
3) NoNanInf (current)
4) NoNanNoInf
5) SupportsNonFinite (if we strongly prefer inverted conditions and this naming)

Please let me know which one looks more reasonable.

Reply:
> FiniteOnly
I find this one most descriptive.

Reply:
Thank you, I will refresh with this "FiniteOnly".

Reply:
@kuhar , I updated the naming to "FiniteOnly" in the latest revision. Please let me know if it's good to go.